firmware/Core/Inc/unit_tests/test_telecommand_parser.h

```diff
@@ -12,4 +12,6 @@ uint8_t TEST_EXEC__TCMD_get_suffix_tag_uint64();
 
 uint8_t TEST_EXEC__TCMD_ascii_to_double();
 
+uint8_t TEST_EXEC_TCMD_TCMD_parse_full_telecommand();
```

Comment:
Thanks for working on this :) 

Function name should be `TEST_EXEC_TCMD_parse_full_telecommand`